packages/content/templates/nuxt-content.dev.vue

```diff
@@ -98,7 +98,11 @@ export default {
 }
 
 .nuxt-content-editor {
+  box-sizing: border-box;
+  display: block;
   width: 100%;
   padding: 8px;
+  overflow: hidden;
+  resize: none;
```

Comment:
`width: 100% + horizontal padding` means the element overflows parents width.
So I added `box-sizing: border-box`.

Default style of `textarea` is `display: inline-block` but I explicitly set it as `block` (maybe this is just preference).

`overflow: hidden`  is to make situation simple and prevent redundant scroll bar from appearing.

`resize: none` is also to make situation simple. That is, with `overflow: hidden`, if developer shrink textarea vertically, overflown lines are hidden.